x-pack/plugins/synthetics/server/legacy_uptime/lib/requests/get_index_status.ts

```diff
@@ -19,7 +24,21 @@ export const getIndexStatus: UMElasticsearchQueryFn<{}, StatesIndexStatus> = asy
           _shards: { total },
         },
       },
-    } = await uptimeEsClient.count({ terminate_after: 1 });
+    } = await uptimeEsClient.count({
+      terminate_after: 1,
+      ...(to && from
+        ? {
+            query: {
+              range: {
+                '@timestamp': {
+                  gte: from,
+                  lte: to,
+                },
+              },
+            },
+          }
+        : {}),
+    });
```

Comment:
`getIndexStatus` only reads the `_shards.total` to determine if index exists. The time range parameter won't have any impact in this case as the `count` property is not being considered.

Reply:
@shahzad31 quick drive-by comment: if you only care about whether the index exists there might be a quicker way.  _count goes to the shards but whether the index exists should be available on the coordinating node I think. If you add a time range anyway and don't have a huge amount of indices it probably doesn't matter, but looks like this call also allows non time range bound queries, which are a risk anyway because they might hit cold or frozen nodes